src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+

```

Comment:
Redundant empty line after the package declaration. Please remove it to adhere to the checklist.